### PR TITLE
added wind strength description

### PR DESCRIPTION
**Pull request by Cutler7**

### CODE REVIEW

File: src/components/RainWindInfo/WindDescription.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  { min: 17, max: 20.799, label: "storm" },
+  { min: 20.8, max: 24.499, label: "strong storm" },
+  { min: 24.5, max: 29.999, label: "violent storm" },
+  { min: 30, max: 1000, label: "hurricane" }
+];
+
+export const WindDescriptionWrapper = styled.span``;
+
+export const WindDescription = ({ windSpeed, className }: WindDescriptionProps): JSX.Element => {
+  const item = WindSpeedValues.find(({ min, max }) => windSpeed >= min && windSpeed < max);
```
Comment (Duelsik) [inline src/components/RainWindInfo/WindDescription.tsx:29]: If you are doing stuff that depends on single props value AND it is something that requires more computing power than transforming a string or adding couple numbers - generally it is a good idea to use React.useMemo.

In this case performance gain will be minimal, but generally "by convention" I will ask to change that.
This line should be:
```tsx
  const item = React.useMemo(
    () => WindSpeedValues
      .find(({ min, max }) => windSpeed >= min && windSpeed < max),
    [ windSpeed ]
  );
```

File: src/components/RainWindInfo/WindDescription.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  const { label } = item || WindSpeedValues[0];
```
Comment (Duelsik) [inline src/components/RainWindInfo/WindDescription.tsx:30]: Above the component (below declaring WindSpeedValues do: `const defaultWindSpeedValue = arrZero(WindSpeedValues);`
Then in this place `const { label } = item || defaultWindSpeedValue;`.

**Review by Duelsik — CHANGES_REQUESTED** [review]
Looks okay, just details to polish.

File: src/components/RainWindInfo/WindDescription.tsx (hunk 1 above)
Comment (Cutler7) [reply src/components/RainWindInfo/WindDescription.tsx:29]: done

**Review by Duelsik — APPROVED** [review]

**Pull request merged**